core-bundle/contao/themes/flexible/styles/layout/list-view.pcss

```diff
@@ -393,7 +411,6 @@
     font-size: .815rem;
     display: flex;
     align-items: center;
-    border-top-left-radius: var(--border-radius);
```

Comment:
Removing this would break the layout:

<img width="304" height="197" alt="" src="https://github.com/user-attachments/assets/03e7b666-a422-44a3-8378-87d1a585a19e" />

Reply:
not sure why that was in there… removed in 1536e936eb1b38a19e195285dfea812e75db958d